pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.black]
-target-version = ['py37', 'py38', 'py39', 'py310']
+target-version = ['py39', 'py310', 'py311', 'py312', 'p313']
 
 [tool.mypy]
-python_version = 3.7
+python_version = 3.13
```

Should this be the lowest-supported Python or the highest-supported Python?
```suggestion
python_version = 3.9
```